src/main/java/nl/Tech/It/Easy/Tech/It/Easy/controllers/Televisions.java

```diff
@@ -0,0 +1,105 @@
+package nl.Tech.It.Easy.Tech.It.Easy.controllers;
+
+
+public class Televisions {
+    private double height;
+    private int width;
+    private String schermKwaliteit;
+    private boolean wifi;
+    private boolean smartTv;
+    private boolean voiceControl;
+    private boolean hdr;
+    private int id;  // Changed to int for ID
```

Comment:
Een int voor ID is prima, het type maakt in feite niet uit. Het mag ook Long of UUID of zelfs String zijn, als het maar uniek is per instance (zoals de Unique Identifier uit een database tabel)